be/src/Project/Services/Catalog/Catalog.Api/ServicesModels/Categories/CreateCategoryServiceModel.cs

```diff
@@ -7,8 +8,7 @@ namespace Catalog.Api.ServicesModels.Categories
     public class CreateCategoryServiceModel : BaseServiceModel
     {
         public string Name { get; set; }
-        public string Schema { get; set; }
-        public string UiSchema { get; set; }
+        public IEnumerable<CategorySchema> Schemas { get; set; }
```

Comment:
**suggestion:** Change to CategorySchemaServiceModel instead of CategorySchema. It is service model. Not Enittiy.